packages/node_modules/@webex/internal-plugin-calendar/test/unit/spec/calendar.js

```diff
@@ -0,0 +1,90 @@
+/*!
+ * Copyright (c) 2015-2019 Cisco Systems, Inc. See LICENSE file.
+ */
+
+import {assert} from '@webex/test-helper-chai';
+import Calendar from '@webex/internal-plugin-calendar';
+import MockWebex from '@webex/test-helper-mock-webex';
+import sinon from 'sinon';
+
+describe('internal-plugin-meetings', () => {
+  beforeEach(() => {
+    // LoggerConfig.set({verboseEvents: true, enable: false});
+    // LoggerProxy.set(logger);
+    // TriggerProxy.trigger = sinon.stub().returns(true);
+    // Metrics.initPayload = sinon.stub();
+    // Metrics.postEvent = sinon.stub();
+    // MediaUtil.createPeerConnection = sinon.stub().returns({});
+  });
+  describe('meeting index', () => {
+    const webex = new MockWebex({
+      children: {
+        calendar: Calendar
+      }
+    });
+
+    beforeEach(() => {
+      webex.canAuthorize = true;
+      webex.internal.device = {
+        register: sinon.stub().returns(Promise.resolve()),
+        unregister: sinon.stub().returns(Promise.resolve())
+      };
+      webex.internal.mercury = {
+        connect: sinon.stub().returns(Promise.resolve()),
+        disconnect: sinon.stub().returns(Promise.resolve()),
+        on: sinon.spy(),
+        off: sinon.spy()
+      };
+
+      webex.request = sinon.stub().returns(Promise.resolve({body: {items: [{id: 'calendar1'}, {id: 'calendar2'}]}}));
+    });
+    describe('Public Api Contract', () => {
+      describe('#register', () => {
```

Comment:
For consistency sake:
```suggestion
      describe('#register()', () => {
```